projects/packages/search/src/dashboard/components/pages/dashboard-page.jsx

```diff
@@ -68,10 +68,19 @@ export default function DashboardPage( { isLoading = false } ) {
 
 	// Introduce the gate for new pricing with URL parameter `new_pricing_202208=1`
 	const isNewPricing = useSelect( select => select( STORE_ID ).isNewPricing202208(), [] );
-	const isJustUpgraded = useSelect(
-		select => select( STORE_ID ).isFeatureEnabled( 'just_upgraded' ),
-		[]
-	);
+
+	const myStorage = window.localStorage;
+	const isTooltipRead = myStorage.getItem( 'upgrade_tooltip_finished' );
```

Comment:
Would it be better to cache the value with `useMemo`, otherwise it would be called on every re-render.

Reply:
Nice catch!